### PR TITLE
Remove FromString method of enumerations

### LINKED ISSUE #98: Remove `FromString` method of enumerations
After https://github.com/nspcc-dev/neofs-sdk-go/issues/86 changes will be released deprecated `FromString` method should be removed. The implementations of `fmt.Stringer` on same types should be simplified after the removal (search TODO with `neofs-api-go#346` pattern).

### PR DESCRIPTION
**Pull request by smallhive**
close #98

### CODE REVIEW

**Comment by roman-khimov** [comment]
Do we want to completely remove `String()`? It can be useful in some cases.

**Comment by cthulhu-rider** [comment]
> Do we want to completely remove `String()`? It can be useful in some cases.

they must be saved, issue #98 didn't mention their removal

File: storagegroup/storagegroup.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ func (sg *StorageGroup) UnmarshalJSON(data []byte) error {
 // error if object is not of TypeStorageGroup type.
 func ReadFromObject(sg *StorageGroup, o objectSDK.Object) error {
 	if typ := o.Type(); typ != objectSDK.TypeStorageGroup {
-		return fmt.Errorf("object is not of StorageGroup type: %s", typ)
+		return fmt.Errorf("object is not of StorageGroup type: %T", typ)
```
Comment (cthulhu-rider) [inline storagegroup/storagegroup.go:255]: we actually wanna see the value of `typ`. If fmt.Stringer is not provided lets use `%v` notation

**Review by cthulhu-rider — APPROVED** [review]

File: storagegroup/storagegroup.go (hunk 1 above)
Comment (smallhive) [reply storagegroup/storagegroup.go]: replaced with %v

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**